src/utils/loadDataset.js

```diff
@@ -19,6 +19,8 @@ export default async dpID => {
   try {
     const dataset = await Dataset.load(DP_ID)
 
+    // console.log('dataset: ' + JSON.stringify(dataset, null, 2))
+
```

Comment:
do we need the comment or can be cleaned up?

Reply:
:+1: cleaned up